### ISSUE TITLE
Unable to boot mojave installer

### ISSUE THREAD

**Issue by d-jahren** [tylernguyen/x1c6-hackintosh#1]
I've followed the instructions and I unfortunately get a kernel panic relating to the Kaby Lake Intel GPU driver framebuffer when I try to boot the installer.

Any suggestions?

Thank you for the help!

**Comment by tylernguyen** [comment]
Hello,

I'm very sorry for the late reply. Are you still facing this issue?

**Comment by foxevereg** [comment]
Have u ever tried an external monitor?

**Comment by Porco-Rosso** [comment]
Hi @d-jahren and @tylernguyen,
I am running into the same problem as you d-jahren. 

I had to make sure to add HFSPlus.efi to see the installer.
The installer crashes while loading, as happened to you.
however editing the gpu code has not changed anything. I've run the installer in verbose mode, and it seems to have a kernal panic and fail related to some intel gpu drivers.
I've tried to look at the information in this guide, but so far nothing has worked.
Would love some hints or a point into the right direction.

**Comment by Porco-Rosso** [comment]
I eventually got to the MacOs installer by using the following kexts, and the config_HD615_620_630_640_650.plist

kexts (clover): 
- DatahubDXe
- FSinject
- SMCHelper
- APFSDRiverloader
- AptioMemoryfix
- EmuVariableUefi
- NVmexpress

(Other) Kexts:
Fake SMC
Lilu
USBinjectAll
VoodooPS2Controller
WhateverGreen

Unfortunately it seems even with the latest Mojave version the PM981 NVMe drive does not work. (crashes during the "install" phase). Shame, cause the PM981 is a great drive, was really hoping by now it would be supported somehow

**Comment by d-jahren** [comment]
@Porco-Rosso Yeah I ran into that as well. From my experience it's better to use older hardware for experimenting with hackintosh and just purchase a real mac if OS X is required for daily use.

**Comment by tylernguyen** [comment]
@Porco-Rosso The PM981 is not compatible. That's why I replaced my drive in the guide.
